app/config/variables.php

```diff
@@ -440,7 +440,7 @@
         'variables' => [
             [
                 'name' => '_APP_SMS_PROVIDER',
-                'description' => "Provider used for delivering SMS for Phone authentication. Use the following format: 'sms://[USER]:[SECRET]@[PROVIDER]'.\n\nEnsure `[USER]` and `[SECRET]` are URL encoded if they contain any non-alphanumeric characters.\n\nAvailable providers are twilio, text-magic, telesign, msg91, and vonage.",
+                'description' => "Provider used for delivering SMS for Phone authentication. Use the following format: 'sms://[USER]:[SECRET]@[PROVIDER]'.\n\nEnsure `[USER]` and `[SECRET]` are URL encoded if they contain any non-alphanumeric characters.\n\nAvailable providers are twilio, text-magic, telesign, msg91, vonage and geosms. When using the geosms provider, the adapter will parse `default` query parameter for a nested provider unless a `local-x` parameter is set for a country calling code. `default` and `local-x` parameters should be URL encoded.",
```

Comment:
This is very confusing to read, any ideas @gewenyu99?

Reply:
Had another go